starknet/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@hyperlane-xyz/starknet-core",
```

Comment:
you'll need to ensure the Dockerfile knows about this new package too

https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/0cd15ae26e9348d7c13132daf2bb6c7606509082/Dockerfile#L24

ie just add after the solidity line
```Dockerfile
 COPY starknet/package.json ./starknet/ 
```